README.md

```diff
@@ -0,0 +1,25 @@
+
+# Applay a Patch Action
+
+This action helps to apply a patch that from a compared commits generated by Github. 
```

Comment:
Can you please run this entire file through a grammar/orthography check tool? There are multiple typos and grammar issues here. Please do a proof-read as well.